audit-ci.jsonc

```diff
@@ -102,6 +107,13 @@
     // from: @arbitrum/nitro-contracts>hardhat>semver
     // from: arb-bridge-peripherals>arb-bridge-eth>hardhat>semver
     // Used only in dev.
-    "GHSA-c2qf-rxjj-qqgw"
+    "GHSA-c2qf-rxjj-qqgw",
+    // https://github.com/advisories/GHSA-g4vp-m682-qqmp
+    // contracts using ERC2771Context along with a custom trusted forwarder may see
+    // _msgSender return address(0) in calls that originate from the forwarder with calldata shorter than 20 bytes
+    // from: @arbitrum/nitro-contracts>@openzeppelin/contracts-upgradeable
+    // from @arbitrum/nitro-contracts>@openzeppelin/contracts
+    //
```

Comment:
need some brief explanation why it's safe to ignore

Reply:
we are not affected (not using that particular contract) and also it does not affect the sdk anyway